### ISSUE TITLE
Living Dex not generating Scatterbug nor Spewpa in Omega Ruby

### ISSUE THREAD

**Issue by yor-so** [architdate/PKHeX-Plugins#227]
**Describe the bug**
Generating a living dex is not generating Scatterbug nor Spewpa in Omega Ruby. I tried with different plugin settings and I can't get it to work.

**To Reproduce**
-Open a Omega Ruby save.
 -Generate Living Dex

**Expected behavior**
Scatterbug nor Spewpa should be generated right before Vivillion.

**Screenshots**
![image](https://github.com/architdate/PKHeX-Plugins/assets/5457965/fa004199-e9c6-433a-9ede-203e765bcaf1)

**Comment by yor-so** [comment]
I did some debugging and it seems that generated sets are invalid/illegal, that's why those pokemon are not being generated/returned. Not sure if the problem is in pkhex or in ALM.
I changed the code locally to return the pkm even if it is invalid (see screenshot attached). ALM is not able to legalize them because there's no legal mons that matches their traits, so I imported a showdown set in order to make them legal and at that point they're good. 

In any case, if invalid/illegal sets are generated like in this case, maybe an error message should be displayed? Or maybe ALM should have a setting to return those invalid sets, so users can at least legalize them on their own, like I did?

![image](https://github.com/architdate/PKHeX-Plugins/assets/5457965/61b5eb6c-1fe5-4deb-8030-35da9315a2c1)
